### PR TITLE
OSD-14684: Add Backplane cloud console and credentials commands

### PR DESCRIPTION
**Pull request by samanthajayasinghe**
## Introduction 
This PR adds the following things to backplane cli 

* Add cloud console command 
```
ocm backplane cloud console <cluster-id>
```

* Add cloud credentials command 
```
ocm backplane cloud credentials <cluster-id>
```

## Reference 
[OSD-14684](https://issues.redhat.com/browse/OSD-14684)

### CODE REVIEW

File: cmd/ocm-backplane/cloud/console.go [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+		"browser",
+		"b",
+		false,
+		fmt.Sprintf("Open a browser after the console container starts. Can also be set via the environment variable '%s'", EnvBrowserDefault),
+	)
+	flags.StringVar(
+		&consoleArgs.backplaneURL,
+		"url",
+		"",
+		"Specify backplane url. Default: The corresponding hive shard of the target cluster.",
```
Comment (wanghaoran1988) [inline cmd/ocm-backplane/cloud/console.go:68]: "URL of backplane API"

Comment (samanthajayasinghe) [reply cmd/ocm-backplane/cloud/console.go]: Thanks, @wanghaoran1988 , 
I fixed both command's URL descriptions.

**Pull request merged**